_posts/2023-09-13-prospero-add-feature-pack.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title:  "Installing additional feature packs in WIldfly"
+date:   2023-09-13
+tags:   prospero galleon wildfly
+author: spyrkob
+description: Using Prospero to installing additional feature packs in existing Wildfly servers.
+---
+
```

Comment:
@spyrkob you're adding great content describing Prospero. We should provide a breadcrumb across all the articles.

For example, you could add a short introduction that starts with: 

```
In [https://www.wildfly.org/news/2023/04/05/prospero/](https://www.wildfly.org/news/2023/04/05/prospero/), you learnt how to install and use [https://github.com/wildfly-extras/prospero](Prospero) to provision a WildFly server. In this article, you will learn how to use Prospero to modify an existing server and extends its capabilities by adding a MariaDB datasource.
```